packages/untaped-awx/src/untaped_awx/application/get_resource.py

```
@@ -2,30 +2,32 @@
 
 class GetResource:
     def __init__(self, client: ResourceClient) -> None:
         self._client = client
 
     def __call__(
         self,
-        spec: ResourceSpec,
+        spec: AwxResourceSpec,
         *,
         name: str | None = None,
         id: int | None = None,
```

Ruff (0.15.12)

[error] Function argument `id` is shadowing a Python builtin

(A002)